README.md

```diff
@@ -1,19 +1,218 @@
 # **Allstar**
 
-GitHub App installed on organizations or repositories to set and enforce security policies.
+Allstar is a GitHub App installed on organizations or repositories to set and
+enforce security policies. It's goal is to be able to detect any GitHub setting,
+or repository file contents that may be risky or not follow security best
+practices. Then, if Allstar finds a repository to be out of compliance, it will
+take an action such as create an issue.
 
-##
-**Status**
+The specific policies are intended to be highly configurable, to try to meet the
+needs of different project communities and organizations. Also, developing and
+contributing new policies is intended to be easy as well.
 
-Currently in early prototype phase. Working on finishing code for "mvp" and documentation for running and contributing policies.
+Allstar is developed under the [OpenSSF](https://openssf.org/) and as a part of
```

Comment:
[OpenSSF](https://openssf.org/) organization, as part of

Reply:
done